Makefile

```diff
@@ -30,35 +30,36 @@ install:
 install_c:
 	CGO_ENABLED=1 go install $(BUILD_FLAGS) -tags "$(BUILD_TAGS) cleveldb" ./cmd/tendermint
 
-########################################
-### Protobuf
+###############################################################################
+###                                Protobuf                                 ###
+###############################################################################
 
-protoc_all: protoc_libs protoc_merkle protoc_abci protoc_grpc protoc_proto3types
+proto-all: proto-gen proto-lint proto-check-breaking
 
-%.pb.go: %.proto
-	## If you get the following error,
```

Comment:
should we preserve this instruction somewhere? (I mostly found it annoying and never actually ran into this issue, but I assume it's here for a reason.)

Reply:
left it where it is for now.